tools/sim/bridge/carla.py

```diff
@@ -0,0 +1,171 @@
+import numpy as np
+from openpilot.common.params import Params
```

Comment:
breakup the openpilot imports with a new line

Reply:
done